hugegraph-core/src/main/java/com/baidu/hugegraph/backend/cache/CachedGraphTransaction.java

```diff
@@ -394,8 +394,13 @@ protected final void commitMutation2Backend(BackendMutation... mutations) {
                 }
             }
 
-            // Update edge cache if any edges change
-            if (edgesInTxSize > 0 && this.enableCacheEdge()) {
+            /*
+             * Update edge cache if any vertex or edge changed
+             * For vertex change, the edges linked with should also be updated
+             * Before we use a more precise strategy,now we just clear all the edge cache
```

Comment:
enhance to--> Before we find a more precise strategy, just clear all the edge cache now (expect a space after `,`)

Reply:
done